packages/amplication-client/.env

```diff
@@ -1,6 +1,6 @@
 NX_REACT_APP_FEATURE_FLAGS="{}"
 NX_REACT_APP_DATA_SOURCE="http://localhost:3000/graphql"
-NX_REACT_APP_PLUGIN_API_DATA_SOURCE="http://localhost:3005/graphql"
+NX_REACT_APP_PLUGIN_API_DATA_SOURCE="https://plugin-api.amplication.com/graphql"
 NX_REACT_APP_STORAGE_GATEWAY_URL="http://localhost:3002"
```

Comment:
This was already fixed in `master`. Resolving the conflict will remove this change @yuval-hazaz